eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -19,6 +19,7 @@ parameters:
   # Multiple labels seperated by comma, e.g. "bug, APIView"
   PRLabels: ''
   SkipCheckingForChanges: false
+  IsTest: false
```

Comment:
Perhaps we should call this what it does. So something like `CloseAfterOpenForTesting`